src/sentry/runner/commands/configoptions.py

```diff
@@ -77,10 +77,13 @@ def _attempt_update(
     is_flag=True,
     help="Hide the actual value of the option on DB when detecting drift.",
 )
+@click.option("--timestamp", help="Timestamp to measure latency for the automator.")
```

Comment:
What does the timestamp that will be passed here represent exactly? Presumably it's something different / earlier to simply when this function is being called?

Reply:
did you want to define a type here, to validate that it isn't being called with a value that isn't a valid timestamp?